### PR TITLE
Update to latest v1 Flask

### PR DESCRIPTION
**Pull request by bjgill**
https://trello.com/c/obh5gpQp/2250-update-to-flask-114

Stop pinning to v1.0.x, since v1.1.x was recently released. We're not ready to move to v2, but do want to be on the most recent v1.

Flask now controls the version of wekzeug, so we no longer need to pin wekzeug ourselves.

### CODE REVIEW

**Comment by bjgill** [comment]
Ok. Tests should be working now. But the breakage suggests that this should be a major version bump

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,13 @@
 # Digital marketplace utils changelog
 
-Records breaking changes from major version bumps
+Records breaking changes from major version bumps. This has some known 
+[breaking changes](https://flask.palletsprojects.com/en/2.0.x/changes/#version-1-1-0).
```
Comment (alex9smith) [inline CHANGELOG.md:4]: Should this be on line 10 instead?

**Comment by alex9smith** [comment]
A major change makes sense to me. Hopefully this doesn't break too many things in the apps that use utils and create a load of extra work for us.

File: CHANGELOG.md (hunk 1 above)
Comment (bjgill) [reply CHANGELOG.md:4]: Good catch.

**Review by alex9smith — APPROVED** [review]
👍

**Pull request merged**